tox.ini

```diff
@@ -1,10 +1,48 @@
 [tox]
-envlist = py3-{lint,test}
+envlist =
+    fix,
+    lint,
+    test
 skipsdist = true
-sitepackages = true
+
+[base]
+deps =
+    -rrequirements.txt
+
+[base-test]
+deps =
+    -rrequirements-test.txt
```

Comment:
And this `requirements-test.txt`

Reply:
No such file exists in the upstream repository at https://github.com/SUSE/prometheus-webhook-snmp so I'm not sure this is a problem for us to resolve here.